src/components/NavBar/Navbar.jsx

```diff
@@ -1,8 +1,43 @@
 import React from 'react';
 import './Navbar.css';
+import NavLogo from '../../Images/cycle-favicon.png';
+import ProfileImg from '../../Images/pngwing.com.png';
```

Comment:
instead of importing file from its individual path. You can make common export index.js file as you made in components.